src/storage/mod.rs

```diff
@@ -1068,9 +1082,9 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
         cmd: TypedCommand<T>,
         callback: Callback<T>,
     ) -> Result<()> {
-        if self.enable_ttl {
+        if self.api_version == ApiVersion::V1ttl {
```

Comment:
This may conflict with `CAS with TTL`.

Reply:
This will be solved in https://github.com/tikv/tikv/pull/11216